script/create_job

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd "$(dirname "$0")/.."
+
+exec python3 -m docker_etl.create_new_job "$@"
```

Comment:
Why call exec instead of python3 directly?

Reply:
I don't think it makes any tangible difference but I saw that bigquery-etl does it so I just did it for consistency.  I forgot to ask about it.  Might be better to not have it so we can add things in the future without having to remove the exec.